[2024-07-16 02:46:49,801] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-11-08T00:00:00+00:00 [queued]>
[2024-07-16 02:46:49,974] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-11-08T00:00:00+00:00 [queued]>
[2024-07-16 02:46:49,977] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:49,978] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:49,979] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:50,051] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-11-08T00:00:00+00:00
[2024-07-16 02:46:50,083] {standard_task_runner.py:52} INFO - Started process 3705 to run task
[2024-07-16 02:46:50,089] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-11-08T00:00:00+00:00', '--job-id', '1029', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpha3ky8q9', '--error-file', '/tmp/tmp6xnuhgap']
[2024-07-16 02:46:50,095] {standard_task_runner.py:77} INFO - Job 1029: Subtask second_task
[2024-07-16 02:46:50,283] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-11-08T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:50,450] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-08T00:00:00+00:00
[2024-07-16 02:46:50,451] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:50,461] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:50,487] {bash.py:169} INFO - Output:
[2024-07-16 02:46:50,490] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:50,491] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:50,615] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211108T000000, start_date=20240716T024649, end_date=20240716T024650
[2024-07-16 02:46:50,896] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:50,933] {local_task_job.py:146} INFO - Task exited with return code 0
